cmd/envtool/main.go

```diff
@@ -92,35 +93,13 @@ func waitForPort(ctx context.Context, port uint16) error {
 		sleepCancel()
 	}
 
-	return ctx.Err()
-}
-
-func waitForPostgresPort(ctx context.Context, port uint16) error {
-	logger := zap.S().Named("postgres.wait")
-
-	for ctx.Err() == nil {
-		args := fmt.Sprintf(`exec -T postgres psql -U postgres -d ferretdb -h 127.0.0.1 --port %d --quiet --command select`, port)
-		if err := tryCompose(strings.Split(args, " "), nil, logger); err == nil {
-			return nil
-		}
-
-		sleepCtx, sleepCancel := context.WithTimeout(ctx, time.Second)
-		<-sleepCtx.Done()
-		sleepCancel()
-	}
-
-	return ctx.Err()
+	return fmt.Errorf("failed to connect to port %d", port)
```

Comment:
We should include a full address ("127.0.0.1:%d") as the common networking problem is about bad Docker network configuration

Reply:
Fixed